addons/account_peppol/models/res_partner.py

```diff
@@ -3,57 +3,68 @@
             ('not_valid_format', 'Cannot receive this format'),  # registered on Peppol but cannot receive the selected document type
             ('valid', 'Valid'),
         ],
-        string='Peppol endpoint validity',
-        compute='_compute_account_peppol_verification_label',
-        copy=False,
-    )  # field to compute the label to show for partner endpoint
+        string='Peppol endpoint verification',
+        default='not_verified',
+        company_dependent=True,
+    )
     is_peppol_edi_format = fields.Boolean(compute='_compute_is_peppol_edi_format')
 
-    def _auto_init(self):
-        """Create columns `account_peppol_is_endpoint_valid` and `account_peppol_validity_last_check`
-        to avoid having them computed by the ORM on installation.
-        """
-        if not column_exists(self.env.cr, 'res_partner', 'account_peppol_is_endpoint_valid'):
-            create_column(self.env.cr, 'res_partner', 'account_peppol_is_endpoint_valid', 'boolean')
-        if not column_exists(self.env.cr, 'res_partner', 'account_peppol_validity_last_check'):
-            create_column(self.env.cr, 'res_partner', 'account_peppol_validity_last_check', 'timestamp')
-        return super()._auto_init()
-
     # -------------------------------------------------------------------------
-    # BUSINESS ACTIONS
+    # HELPERS
     # -------------------------------------------------------------------------
 
+    def _log_verification_state_update(self, company, old_value, new_value):
+        # log the update of the peppol verification state
+        # we do this instead of regular tracking because of the customized message
+        # and because we want to log the change for every company in the db
+        if old_value == new_value:
+            return
+
+        peppol_verification_state_field = self._fields['peppol_verification_state']
+        selection_values = dict(peppol_verification_state_field.selection)
+        old_label = selection_values[old_value] if old_value else False  # get translated labels
+        new_label = selection_values[new_value] if new_value else False
+
+        body = Markup("""
+            <ul>
+                <li>
+                    <span class='o-mail-Message-trackingOld me-1 px-1 text-muted fw-bold'>{old}</span>
+                    <i class='o-mail-Message-trackingSeparator fa fa-long-arrow-right mx-1 text-600'/>
+                    <span class='o-mail-Message-trackingNew me-1 fw-bold text-info'>{new}</span>
+                    <span class='o-mail-Message-trackingField ms-1 fst-italic text-muted'>({field})</span>
+                    <span class='o-mail-Message-trackingCompany ms-1 fst-italic text-muted'>({company})</span>
+                </li>
+            </ul>
+        """).format(
+            old=old_label,
+            new=new_label,
+            field=peppol_verification_state_field.string,
+            company=company.display_name,
+        )
+        self._message_log(body=body)
```

Comment:
It's not a good idea to save static templating inside message body. Classnames / html tags and related style can change in the future, so these messages will have broken style sooner or later.

I suppose the intent was for message to look like tracking values without using tracking values (because they are not really). One solution could have been to store pure data somewhere (could be inside message body with data that are ok with sanitizer, e.g. `data-oe-type` and so on), and then in client-side code (JS / XML / SCSS) to retrieve and render these data like tracking values.